CHIRPS_precipitationNiger.R

```diff
@@ -0,0 +1,379 @@
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Author: Catherine Back and Elinor Benami
+# Project name: Sahel Adaptive Social Protection Program
+# Date Created: # Thu Jun 30 08:26:20 2022 ------------------------------
+# Date Last Updated:
+# R version: 4.1.3
+# Purpose: Load and Clean Precip Data Files
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Load Libraries and Options -----
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+library(tidyverse)
+library(tidylog)
+
+library(openxlsx)
+library(readxl)
+
+library(readr)
+library(tidyr)
+library(lubridate)
+
+library(ggplot2)

```

Comment:
tidyverse includes several of the packages listed below, so don't need to call them explicitly. In particular, tidyverse includes the following:

[ggplot2](https://ggplot2.tidyverse.org/), for data visualisation.
[dplyr](https://dplyr.tidyverse.org/), for data manipulation.
[tidyr](https://tidyr.tidyverse.org/), for data tidying.
[readr](https://readr.tidyverse.org/), for data import.
[purrr](https://purrr.tidyverse.org/), for functional programming.
[tibble](https://tibble.tidyverse.org/), for tibbles, a modern re-imagining of data frames.
[stringr](https://github.com/tidyverse/stringr), for strings.
[forcats](https://github.com/tidyverse/forcats), for factors.

would recommend removing explicit calls of those packages